web-app/src/components/HolidayModal/container.js

```diff
@@ -1,16 +1,25 @@
 import React, { Component } from 'react';
 import { PropTypes as PT } from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
 import { isEmpty } from 'lodash';
 import { approveHoliday, rejectHoliday } from '../../services/holidayService';
 import swal from 'sweetalert2';
 import holidayStatus from '../../utilities/holidayStatus';
 import { Toast } from '../../utilities/Notifications';
+import { getUser } from '../../reducers';
 
-export default Wrapped =>
+const HolidayModalContainer = Wrapped =>
   class extends Component {
     static propTypes = {
       holiday: PT.object.isRequired,
       closeModal: PT.func.isRequired,
+      showAdminControls: PT.bool,
+      userDetails: PT.object.isRequired,
+    };
+
+    static defaultProps = {
+      showAdminControls: true,
```

Comment:
should the default here not be `false`? - I appreciate the default is unlikely to ever be hit but as a fallback I assume you don't want to show admin controls